exporter/collector/googlecloud.go

```diff
@@ -45,6 +48,20 @@ func setVersionInUserAgent(cfg *Config, version string) {
 	cfg.UserAgent = strings.ReplaceAll(cfg.UserAgent, "{{version}}", version)
 }
 
+func setProjectFromADC(ctx context.Context, cfg *Config, scopes []string) error {
```

Comment:
How many places do we have this code now :)

Reply:
Just one in the collector exporter, which is now here.